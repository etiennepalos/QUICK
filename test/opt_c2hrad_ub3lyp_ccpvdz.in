UDFT B3LYP BASIS=cc-pVDZ cutoff=1.0e-9 denserms=1.0e-6 MULT=2 OPTIMIZE=3 DIPOLE

C 0.000000 0.000000 0.600000
C 0.000000 0.000000 -0.600000
H 0.000000 0.000000 1.600001
