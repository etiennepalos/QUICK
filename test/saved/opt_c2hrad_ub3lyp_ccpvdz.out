  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2023                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **             This software is subject to the terms of the Mozilla Public License, v. 2.0.            **
  **                                                                                                     **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Shajan, A.; Smith, J.; Miao, Y.; He, X.; Ayers, K.; Brothers, E.; Goetz, A.W.; Merz, K.M.
  QUICK-23.08 University of California San Diego, CA and Michigan State University, East Lansing, MI, 2023 
                                                                                                           
  If you have any comments or queries, please reach out to us on GitHub issues or discussion sections:     
  https://github.com/merzlab/QUICK.                                                                        
                                                                                                           
| TASK STARTS ON: Wed Feb 28 10:06:35 2024
| INPUT FILE :    opt_c2hrad_uhf_ccpvdz.in
| OUTPUT FILE:    opt_c2hrad_uhf_ccpvdz.out
| DATA FILE  :    opt_c2hrad_uhf_ccpvdz.dat
| BASIS SET PATH: /Users/andi/projects/quick/QUICK/install-test/basis

 @ Read Job And Atom

  KEYWORD=UDFT B3LYP BASIS=CC-PVDZ CUTOFF=1.0E-9 DENSERMS=1.0E-6 MULT=2 OPTIMIZE=3 DIPOLE                                                                                                                         
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 USING LIBXC VERSION: 4.3.4
 FUNCTIONAL INFORMATION:
 NAME = B3LYP FAMILY = HYBRID GGA KIND = EXCHANGE CORRELATION
 DIRECT SCF 
 SAD INITAL GUESS 
 UNRESTRICTED SYSTEM
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 DIPOLE
 STANDARD GRID = SG1
 GEOMETRY OPTIMIZATION ANALYTICAL GRADIENT
 REQUEST CRITERIA FOR GEOMETRY CONVERGENCE: 
      MAX ALLOWED GEO CHANGE  =  0.189E+00
      MAX GEOMETRY CHANGE     =  0.180E-02
      GEOMETRY CHANGE RMS     =  0.120E-02
      MAX GRADIENT CHANGE     =  0.100E-02
      GRADIENT NORMALIZATION  =  0.300E-03
      MAX OPTIMIZATION CYCLE  =     3
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE =    3
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-09
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-05
      GRADIENT CUTOFF  =  0.100E-06
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = CC-PVDZ,  TYPE = CARTESIAN
| BASIS FILE = /Users/andi/projects/quick/QUICK/install-test/basis/CC-PVDZ.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = C 
 BASIS FUNCTIONS =   15
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =    5

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    2
 TOTAL ATOM NUMBER       =    3    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    1    NUMBER OF NON-HYDROGEN ATOM =    2
 NUMBER OF ALPHA ELECTRONS =    7
 NUMBER OF BETA ELECTRONS  =    6

 -- INPUT GEOMETRY -- :
    C           0.0000       0.0000       0.6000
    C           0.0000       0.0000      -0.6000
    H           0.0000       0.0000       1.6000

 -- DISTANCE MATRIX -- :
              1           2           3
      1     0.00000
      2     2.26767     0.00000
      3     1.88973     4.15740     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   35
 NSHELL =   15 NPRIM  =   88
 JSHELL =   15 JBASIS =   56


 @ End Reading Molecular Information 


***********************************************************************
**                                                                   **
**                       ---------------------                       **
**                              DL-FIND                              **
**                       Geometry Optimisation                       **
**                       ---------------------                       **
**                                                                   **
**                 J. Kaestner, J.M. Carr, T.W. Keal,                **
**                W. Thiel, A. Wander and P. Sherwood                **
**                                                                   **
**              J. Phys. Chem. A, 2009, 113 (43), 11856.             **
**                                                                   **
**   Please include this reference in published work using DL-FIND.  **
**                                                                   **
**               Copyright:  STFC Daresbury Laboratory               **
**                           Revision:                               **
**                                                                   **
***********************************************************************

Residue member list:
    0 :     1    1    1

Generating new HDLC for    1 residues

Located a new residue for HDLC
   Residue number is    1
   Number of atoms it contains     3
   Atoms:      1     2     3
   Generating HDLC for residue    1
       Making linear about     1

   The Cartesian coordinates
         1     0.00000000000000     0.00000000000000     1.13383567973139
         2     0.00000000000000     0.00000000000000    -1.13383567973139
         3     0.00000000000000     0.00000000000000     3.02356370234316

   The primitive internal coordinates


     The system has     2 stretches
         1 Stre     2     1              =    2.26767136
         2 Stre     3     1              =    1.88972802


     The system has     1 bends and impropers
         3 Line     2     1     3  wrt yz


     The system has     0 dihedrals


   Residue      1 has internal coords      1 to      9

DL-FIND Report:
===============
Optimisation algorithm: L-BFGS
Number of steps in L-BFGS memory ...............        100

Step length: simple scaled
Maximum step length ............................  1.000E-01
Scaling step by ................................  1.000E+00

Coordinate system: Delocalised internal coordinates (DLC)

Number of atoms ................................          3
Number of input geometries .....................          1
Variables to be optimised ......................          3
No restart information is written
This run has not been restarted.
Maximum number of steps ........................          3
Maximum number of energy evaluations ...........     100000


 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   1  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  C           0.000000       0.000000       0.600000
  C           0.000000       0.000000      -0.600000
  H           0.000000       0.000000       1.600001

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11384
|   FINAL GRID POINTS    =        7671
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        5761
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        8616

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         USCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.909616799      ------         0.081     1        0.00      0.08      0.00    0.1779E+01  0.6196E-01  0.4754E+00
|  2    -76.554513900  -.355103E+00       0.090     2        0.00      0.09      0.00    0.1171E+00  0.2206E-01  0.2823E+00
|  3    -76.489641696  -.648722E-01       0.089     3        0.00      0.09      0.00    0.2726E+00  0.1503E-01  0.2064E+00
|  4    -76.593271549  0.103630E+00       0.088     4        0.00      0.09      0.00    0.1013E+00  0.5044E-02  0.6979E-01
|  5    -76.606937214  0.136657E-01       0.086     5        0.00      0.09      0.00    0.5481E-02  0.1365E-02  0.1210E-01
|  6    -76.607370142  0.432928E-03       0.085     6        0.00      0.08      0.00    0.1497E-02  0.1020E-02  0.8701E-02
|  7    -76.607584795  0.214653E-03       0.083     7        0.00      0.08      0.00    0.1702E-02  0.2272E-02  0.2028E-01
|  8    -76.607957222  0.372427E-03       0.085     8        0.00      0.08      0.00    0.1178E-02  0.3958E-02  0.3795E-01
|  9    -76.608217495  0.260274E-03       0.085     9        0.00      0.08      0.00    0.3782E-02  0.2246E-03  0.3478E-02
| 10    -76.608242592  0.250965E-04       0.081    10        0.00      0.08      0.00    0.1604E-02  0.1182E-02  0.1166E-01
| 11    -76.608263855  0.212629E-04       0.082     1        0.00      0.08      0.00    0.1284E-02  0.3285E-03  0.5236E-02
| 12    -76.608273843  0.998826E-05       0.082     2        0.00      0.08      0.00    0.9909E-03  0.1316E-03  0.1297E-02
| 13    -76.608275350  0.150722E-05       0.080     3        0.00      0.08      0.00    0.4840E-03  0.6695E-04  0.8133E-03
| 14    -76.608275902  0.552214E-06       0.078     4        0.00      0.08      0.00    0.1332E-03  0.1849E-04  0.1810E-03
| 15    -76.608275930  0.282472E-07       0.075     5        0.00      0.07      0.00    0.9533E-05  0.1159E-05  0.1125E-04
| 16    -76.608275931  0.543459E-10       0.066     6        0.00      0.07      0.00    0.1081E-05  0.2209E-06  0.2795E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  16 CYCLES
| MAX ERROR = 0.108118E-05   RMS CHANGE = 0.220892E-06   MAX CHANGE = 0.279480E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =    7.0000050554
 BETA ELECTRON DENSITY     =    6.0000179910
 ELECTRONIC ENERGY    =    -97.101862178
 CORE_CORE REPULSION  =     20.493586247
 TOTAL ENERGY         =    -76.608275931

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0000004999
    1Y     0.0000004520
    1Z     0.0426313552
    2X     0.0000001624
    2Y    -0.0000001441
    2Z     0.0273203368
    3X     0.0000003375
    3Y    -0.0000003080
    3Z    -0.0699516920
------------------------
 OPTIMIZATION STATISTICS:
 MAXIMUM GRADIENT ELEMENT=     0.3014492034E-01 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.1740417788E-01 (REQUEST=  0.66667E-03 )

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   2  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  C           0.000000      -0.000000       0.595011
  C          -0.000000       0.000000      -0.597505
  H          -0.000000       0.000000       1.602496

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11384
|   FINAL GRID POINTS    =        7687
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        5516
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        8202

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         USCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.610464662      ------         0.092     1        0.00      0.09      0.00    0.3448E-01  0.6705E-03  0.6653E-02
|  2    -76.608670339  -.179432E-02       0.091     2        0.00      0.09      0.00    0.4180E-02  0.5430E-03  0.8080E-02
|  3    -76.608601457  -.688817E-04       0.083     3        0.00      0.08      0.00    0.9004E-02  0.3721E-03  0.5624E-02
|  4    -76.608696574  0.951166E-04       0.083     4        0.00      0.08      0.00    0.5267E-03  0.5694E-04  0.7062E-03
|  5    -76.608697673  0.109958E-05       0.080     5        0.00      0.08      0.00    0.6040E-04  0.8303E-05  0.1258E-03
|  6    -76.608697698  0.246798E-07       0.075     6        0.00      0.07      0.00    0.6334E-05  0.4225E-05  0.7367E-04
|  7    -76.608697707  0.859050E-08       0.071     7        0.00      0.07      0.00    0.3975E-05  0.3611E-05  0.5999E-04
|  8    -76.608697711  0.404284E-08       0.070     8        0.00      0.07      0.00    0.2218E-05  0.3236E-05  0.5336E-04
|  9    -76.608697712  0.113811E-08       0.070     9        0.00      0.07      0.00    0.3831E-06  0.8507E-07  0.1336E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   9 CYCLES
| MAX ERROR = 0.383071E-06   RMS CHANGE = 0.850656E-07   MAX CHANGE = 0.133626E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =    7.0000081447
 BETA ELECTRON DENSITY     =    6.0000203732
 ELECTRONIC ENERGY    =    -97.178325642
 CORE_CORE REPULSION  =     20.569627930
 TOTAL ENERGY         =    -76.608697712

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0000001121
    1Y     0.0000000906
    1Z     0.0169741062
    2X     0.0000000654
    2Y    -0.0000000004
    2Z     0.0442402534
    3X     0.0000000467
    3Y    -0.0000000902
    3Z    -0.0612143596
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.4217813768E-03 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.2000000000E-01 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.1154700538E-01 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.1200250563E-01 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.6929649855E-02 (REQUEST=  0.66667E-03 )

 @ Finish Optimization for This Step

 @ Optimize for New Step


============  GEOMETRY FOR OPTIMIZATION STEP   3  ============

GEOMETRY INPUT
ELEMENT      X              Y              Z
  C           0.000000      -0.000000       0.591710
  C          -0.000000       0.000000      -0.595855
  H          -0.000000       0.000000       1.604146

 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       11384
|   FINAL GRID POINTS    =        7688
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        5516
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        8198

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         USCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.609915502      ------         0.094     1        0.00      0.09      0.00    0.2287E-01  0.4513E-03  0.4506E-02
|  2    -76.608764400  -.115110E-02       0.092     2        0.00      0.09      0.00    0.2757E-02  0.3576E-03  0.5298E-02
|  3    -76.608735033  -.293673E-04       0.086     3        0.00      0.09      0.00    0.5891E-02  0.2443E-03  0.3678E-02
|  4    -76.608775820  0.407867E-04       0.086     4        0.00      0.09      0.00    0.3427E-03  0.3719E-04  0.4610E-03
|  5    -76.608776285  0.465491E-06       0.084     5        0.00      0.08      0.00    0.4010E-04  0.5452E-05  0.8189E-04
|  6    -76.608776296  0.104307E-07       0.079     6        0.00      0.08      0.00    0.4133E-05  0.2696E-05  0.4688E-04
|  7    -76.608776299  0.343130E-08       0.076     7        0.00      0.08      0.00    0.2529E-05  0.2290E-05  0.3794E-04
|  8    -76.608776301  0.167666E-08       0.076     8        0.00      0.08      0.00    0.1400E-05  0.7649E-06  0.1248E-04
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYCLES
| MAX ERROR = 0.139995E-05   RMS CHANGE = 0.764891E-06   MAX CHANGE = 0.124752E-04
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =    7.0000103192
 BETA ELECTRON DENSITY     =    6.0000221177
 ELECTRONIC ENERGY    =    -97.229593007
 CORE_CORE REPULSION  =     20.620816706
 TOTAL ENERGY         =    -76.608776301

 @ End Energy calculation


 ANALYTICAL GRADIENT: 
------------------------
 VARIBLES    NEW_GRAD
------------------------
    1X    -0.0000000416
    1Y     0.0000000343
    1Z    -0.0002210330
    2X     0.0000000474
    2Y     0.0000000165
    2Z     0.0558383896
    3X    -0.0000000058
    3Y    -0.0000000509
    3Z    -0.0556173566
------------------------
 OPTIMIZATION STATISTICS:
 ENERGY CHANGE           =     0.7858877265E-04 (REQUEST=  0.10000E-05 )
 MAXIMUM GEOMETRY CHANGE =     0.1323143122E-01 (REQUEST=  0.40000E-02 )
 GEOMETRY CHANGE RMS     =     0.7639170378E-02 (REQUEST=  0.26667E-02 )
 MAXIMUM GRADIENT ELEMENT=     0.1562939301E-03 (REQUEST=  0.10000E-02 )
 GRADIENT NORM           =     0.9023634734E-04 (REQUEST=  0.66667E-03 )

 @ Finish Optimization for This Step

 WARNING: REACHED MAX OPT CYCLES. THE GEOMETRY IS NOT OPTIMIZED.
          PRINTING THE GEOMETRY FROM LAST STEP.
============= GEOMETRY INFORMATION (NOT OPTIMIZED) ===========

 OPTIMIZED GEOMETRY IN CARTESIAN
 ELEMENT      X              Y              Z
  C           0.000000      -0.000000       0.591753
  C          -0.000000       0.000000      -0.595876
  H          -0.000000       0.000000       1.604125

 FORCE
 ELEMENT      X              Y              Z
  C           0.000000      -0.000000       0.000117
  C          -0.000000      -0.000000      -0.029548
  H           0.000000       0.000000       0.029431

 MINIMIZED ENERGY =    -76.608776301
===============================================================

 @ Finish Optimization Job


DL-FIND Report:
===============
Optimisation algorithm: L-BFGS
Number of steps in L-BFGS memory ...............        100

Step length: simple scaled
Maximum step length ............................  1.000E-01
Scaling step by ................................  1.000E+00

Coordinate system: Delocalised internal coordinates (DLC)

Number of atoms ................................          3
Number of input geometries .....................          1
Variables to be optimised ......................          3
No restart information is written
This run has not been restarted.
Number of energy evaluations on this processor .          3
Number of steps ................................          3
Number of accepted steps / line searches .......          3


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   C              0.1990            -0.2109
   C             -0.1625             0.0189
   H             -0.0365             0.1919
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
    0.0000   -0.0000    0.6869    0.6869

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     0.311795000(  7.03%)
| DFT GRID OPERATIONS =     0.024605000(  0.55%)
| TOTAL SCF TIME      =     2.707804000( 61.01%)
|       TOTAL OP TIME      =     2.701035000( 60.86%)
|             TOTAL 1e TIME      =     0.002898000(  0.07%)
|             TOTAL 2e TIME      =     2.128013000( 47.95%)
|             TOTAL EXC TIME     =     0.566936000( 12.77%)
|       TOTAL DII TIME      =     0.006765000(  0.15%)
|             TOTAL DIAG TIME    =     0.000137000(  0.00%)
| TOTAL GRADIENT TIME      =     1.362618000( 30.70%)
|       TOTAL 1e GRADIENT TIME      =     0.013436000( 0.36%)
|       TOTAL 2e GRADIENT TIME      =     1.195899000(26.95%)
|       TOTAL EXC GRADIENT TIME     =     0.150673000(  3.40%)
| TOTAL TIME          =     4.437965000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  4.4 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Wed Feb 28 10:06:39 2024
